db/migrate/20220110090243_create_seances.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSeances < ActiveRecord::Migration[7.0]
+  def change
+    create_table :seances do |t|
+      t.has_many :reservation
+      t.belongs_to :movie
+      t.belongs_to :hall
+      t.float :time
```

Comment:
Again here - float. Could you explain your idea how you will represent time of seance in float format?

Reply:
Oh, I thought I changed it to integer, hmm maybe  I did it only in one place. Yeah, I changed my mind and want to use an integer for now.